install_config/install/advanced_install.adoc

```diff
@@ -784,6 +783,38 @@ openshift_examples_modify_imagestreams=true
 openshift_docker_additional_registries=example.com
 ----
 
+[[advanced-install-configuring-registry-location]]
```

Comment:
@mburke5678 this is a duplicate id, which seems to be copied from the previous section. I have changed it to advanced-install-configuring-registry-console.